int32 userID
int32 robotID   # -1 --> No robot specified.
string format   # "wav" on chrome (pcm), "webm" on firefox (opus)
uint8[] data
